airflow/providers/amazon/aws/hooks/emr.py

```diff
@@ -258,7 +258,8 @@ def __init__(self, *args: Any, **kwargs: Any) -> None:
 
     def cancel_running_jobs(self, application_id: str, waiter_config: dict = {}):
         """
-        List all jobs in an intermediate state, cancel them, then wait for those jobs to reach terminal state.
+        List all jobs in an intermediate state, cancel them, then returns the number of
+        cancelled jobs.
```

Comment:
```suggestion
        Cancel jobs in an intermediate state, and return the number of cancelled jobs.
```

We’re working on enforcing this docstring format soon.